libraries/AP_Logger/LoggerMessageWriter.h

```diff
@@ -73,6 +73,25 @@ class LoggerMessageWriter_WriteAllRallyPoints : public LoggerMessageWriter {
     Stage stage = Stage::WRITE_NEW_RALLY_MESSAGE;
 };
 
+#if AC_FENCE
+class LoggerMessageWriter_Write_Polyfence : public LoggerMessageWriter {
+public:
+
+    void reset() override;
+    void process() override;
+
+private:
+    enum Stage {
```

Comment:
```suggestion
    enum class Stage {
```

will show up a bug,  I think....

Reply:
done